src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/net/SubscribeResponseHandlerTest.java

```diff
@@ -0,0 +1,57 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.net;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.grpc.SubscribeResponse;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.net.GRPCConnectionManager;
+import com.google.common.collect.Sets;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SubscribeResponseHandlerTest {
+    private static final String HOST = "127.0.0.1";
+    private static final String NODE = "TEST";
+
+    private SubscriptionManager subscriptionManager;
+    private NodeStateManager nodeStateManager;
+    private SubscribeResponseHandler uut;
```

Comment:
Why do we always name as uut ? :)

Reply:
"unit under test", I'm used to that naming convention from Splunk. I don't feel strongly about it :)